packages/file_selector/file_selector_web/README.md

```diff
@@ -0,0 +1,37 @@
+# file_picker_web
+
+The web implementation of [`file_picker`][1].
+
+**Please set your constraint to `file_picker_web: '>=0.1.y+x <2.0.0'`**
+
+## Backward compatible 1.0.0 version is coming
+The plugin has reached a stable API, we guarantee that version `1.0.0` will be backward compatible with `0.1.y+z`.
+Please use `file_picker_web: '>=0.1.y+x <2.0.0'` as your dependency constraint to allow a smoother ecosystem migration.
+For more details see: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
```

Comment:
Nope, let's not make this promise here :)

Reply:
Fixed, nice catch. PTAL